doc/release-notes/5.7-release-notes.md

```diff
@@ -0,0 +1,109 @@
+
+### Skip Checksum Validation at Publish
+
+When a user requests to publish a dataset, the time taken to complete the publishing process varies based on the dataset/datafile size.
+
+With the additional settings of :DatasetChecksumValidationSizeLimit and :DataFileChecksumValidationSizeLimit, the checksum validation can be skipped while publishing.
+
+If the Dataverse administrator chooses to set these values, it's strongly recommended to have an external auditing system run periodically in order to monitor the integrity of the files in the Dataverse installation.
+
+### Guestbook Response API Update
+
+With this release the Retrieve Guestbook Responses for a Dataverse Collection API will no longer produce a file by default. You may specify an output file by adding a -o $YOURFILENAME to the curl command.
+
+### Dynamic JavaServer Faces Configuration Options
+
+This release includes a new way to easily change JSF settings via MicroProfile Config, especially useful during development.
+See the [development guide on "Debugging"](https://guides.dataverse.org/en/5.7/developers/debugging.html) for more information.
+
+### Enhancements to DDI Metadata Exports
+
+Several changes have been made to the DDI exports to improve support for internationalization and to improve compliance with CESSDA requirements. These changes include:
+
+- Addition of xml:lang attributes specifying the dataset metadata language at the document level and for individual elements such as title and description
+- Specification of controlled vocabulary terms in duplicate elements in multiple languages (in the installation default langauge and, if different, the dataset metadata language)
+
+While these changes are intended to improve harvesting and integration with external systems, they could break existing connections that make assumptions about the elements and attributes that have been changed.
+
+## New JVM Options and DB Settings
+
+- :SendNotificationOnDatasetCreation - A boolean setting that, if true will send an email and notification to users when a Dataset is created. Messages go to those, other than the dataset creator, who have the ability/permission necessary to publish the dataset.
+- :DatasetChecksumValidationSizeLimit - disables the checksum validation while publishing for any dataset size greater than the limit.
+- :DataFileChecksumValidationSizeLimit - Disables the checksum validation while publishing for any datafiles greater than the limit.
+- :CVocConf - A JSON-structured setting that configures Dataverse to associate specific metadatablock fields with external vocabulary services and specific vocabularies/sub-vocabularies managed by that service.
+- :MetadataLanguages - Sets which languages can be used when entering dataset metadata.
```

Comment:
Perhaps this "Sets which languages can be used when entering dataset metadata" idea should be promoted a bit more above as a feature? I just looked again at pull request #7958 and it has this to say:

> the PR adds a new metadata languages setting and a mechanism to let collection admins set the metadata language for new datasets (from the allowed list in the setting). The mechanism does not enforce using a specific language but does indicate the requested language when editing and again when metadata is displayed.

Reply:
Thx, added a major use case for this in https://github.com/IQSS/dataverse/pull/8128/commits/4e4fa010565bb2d5eb107f8e2e09d788b178d334